airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -292,6 +301,15 @@ private WorkerPodToleration workerPodToleration(String tolerationStr) {
         .collect(Collectors.toList());
   }
 
+  @Override
+  public MaxWorkersConfig getMaxWorkers() {
+    return new MaxWorkersConfig(
```

Comment:
Does this mean that the default is for a single worker pod to be able to 'supervise' 25 different jobs at once?

Reply:
Yeah. You can't tune the max workers overall for a single temporal worker node (I only saw it per-workflow). LMK if you have an idea of how to do it better.